docs/guides/ci-cd.md

```diff
@@ -216,7 +216,8 @@ After pushing commits to your repository, GitHub will automatically trigger a ru
 
 You can view the logs from each workflow step as they run. The `Prefect Deploy` step will include output about your image build and push, and the creation/update of your deployment.
 
-```
+
+```bash
```

Comment:
nit: i think we typically use the terminal div class instead of `bash` like [this](https://github.com/PrefectHQ/prefect/blob/main/docs/guides/deployment/overriding-job-variables.md?plain=1#L181)

Reply:
that was my only comment for the output